wger/nutrition/templates/plan/overview.html

```diff
@@ -1,12 +1,14 @@
 {% extends "base.html" %}
-{% load i18n %}
+{% load i18n cache %}
```

Comment:
Why have you deleted `{% load i18n %}`

Reply:
I have not deleted it, I just added cache to it in order to cache the template.